app/models/category.rb

```diff
@@ -0,0 +1,3 @@
+class Category < ApplicationRecord
+  validates :name, :icon, presence: true
+end
```

Comment:
- [ ] A category(group) should belong to a user. Please make sure to include this association.